eks.tf

```diff
@@ -0,0 +1,88 @@
+module "eks" {
+  source          = "terraform-aws-modules/eks/aws"
+  cluster_name    = var.project_name
+  cluster_version = "1.29"
+
+  vpc_id          = aws_vpc.vpc.id
+  subnet_ids      = [aws_subnet.public_subnet1.id,aws_subnet.public_subnet2.id]
+
+  cluster_endpoint_public_access	= true
+  cluster_endpoint_public_access_cidrs = var.external_ips
+
+  access_entries = {
+    # One access entry with a policy associated
+    Admins = {
+      kubernetes_groups = []
+      principal_arn     = "arn:aws:iam::123456789012:role/something"
```

Comment:
This still points to a dummy ARN. Is fixing this part of the criteria for closing issue #2 ?

Reply:
This comment is fairly unrelated from the exercise of using variables in the object names. Just wanted to point it out. I'll approve this PR anyway.

Reply:
fixed it to use the Admin role of the current acounnt